.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -0,0 +1,46 @@
+name: 🐛 Report a bug
+description: '___'
+labels: [bug]
```

Comment:
If it works:

```suggestion
labels: bug
```

Reply:
Yes, `labels` can be either Array or String.